### DISCUSSION TITLE
⚙️ Motor and encoder replacement

### DISCUSSION THREAD

**Discussion by francocipollone** [Ekumen-OS/andino#10] [Ideas]
### Summary

The motor and the encoders we are using are ok, given that they are available locally and they are not expensive.
However correctly setting up the encoders(mechanicaly) isn't trivial. And of course, the precision is low.

### Alternative

Here there is a cheap alternative:

https://www.sparkfun.com/products/16413

![image](https://user-images.githubusercontent.com/53065142/211906234-b76660b4-249f-49fc-b54e-41f0c4240e03.png)

Pros:
 - Better precision
 - Easier to assembly

Cons:
 - External purchase (at least from Argentina)

**Comment by Martin-MT** [comment]
The old motors (https://www.amazon.com/perseids-Chassis-Encoder-Wheels-Battery/dp/B07DNYQ3PX/ref=sr_1_2) had the following specification (from the comments/answers):
* 110 RPM at 3v
* 170 RPM at 4.5v
* 230 RPM at 6v

The new ones (https://www.sparkfun.com/products/16413) have the following specification:
* 90RPM at 4.5V

Both have a gearbox ratio of 1:48, so no difference there, but at least on paper, the speed is a big factor

**Comment by Martin-MT** [comment]
The current motors/chassis can be bought on amazon, on the following links (for reference only):
https://www.amazon.com/YIKESHU-Smart-Chassis-Encoder-Battery/dp/B073VHQT6P/ref=sr_1_1
https://www.amazon.com/perseids-Chassis-Encoder-Wheels-Battery/dp/B07DNYQ3PX/ref=sr_1_2
